tests/Benchmarks/Microsoft.Experimental.Collections/MultiValueDictionaryPerformanceTests.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+using BenchmarkDotNet.Attributes;
+
+namespace Microsoft.Collections.Extensions.Tests
+{
+    public class MultiValueDictionaryPerformanceTests
+    {
+        [Params(1000, 10000, 10000)]
+        public int Size { get; set; }
+
+        public int RandomKey { get; set; }
```

Comment:
In other benchmarks, we tend to use private fields rather than public properties. Any specific reason why you opted for properties instead?

Reply:
I was going to use `[Params]` in this property and that is why I used public. I didn't know if when using `[Params]` it can be public, don't know how benchmark initializes the property with the specified parameters in the attribute.

However, I will update to make the dictionary, list and randomkey private 😄

Reply:
`[Params]` requires it to be either a public property with public setter or public non-readonly field